nasa-veda-platform/dashboard/index.qmd

```diff
@@ -1,21 +1,64 @@
+- Otherwise, you have to rely on a highly customized and expensive service.
+- The VEDA Dashboard set of tools allows you to visualize spatio-temporal data assets within your website, maintaining an interoperable reference to the source data. At the same time, it is completely open source, customizable, and comes without a subscription.
 
-![Dashboard Screenshot](images/dashboard.png)
+### 3. We show that, with cloud-optimized data, a platform can enable science workflows AND communicates to stakeholders
+- Data visualization in web interfaces often relies on copies of original data that were prepared for visualization (even scalable visualization services like NASA WorldView rely on statically produced rendered replicas). Linking the visualization back to original source data, as required for (re)producing scientific analysis, is often not possible or requires connecting a lot of dots.
+- VEDA visualizes data directly off a single authoritative reference that contains data optimized for visualization as well as direct access, avoiding the issue of missing links between source and rendered data. The data a user sees in VEDA has a direct reference to source data (often via STAC).
+Another benefit of dynamic tiling is that users are able to adjust viewing parameters such as color map and value range to focus on, to study details of their interest.
 
-## Contributing
+### 4. Dashboard features are built and optimized for Earth science data but might be useful in other similar applications
+- Spatio-temporal data often comes from Earth observations or Earth system models. However, the principle of integrating dynamically rendered data into a website is valuable for many branches of data-intensive science, such as heliophysics or astrophysics
 
-Please see our docs on [Dashboard Configuration](../contributing/dashboard-configuration/index.qmd).
\ No newline at end of file
+### 5. The VEDA Dashboard is completely open-source
+- The code is open and accessible, and the tool itself proliferates the use of open standards and reproducible data access for open science
+Community involvement will become important to the long-term success of VEDA
+
+### 6. The number of websites that use VEDA UI is growing, each with different content and data, site layouts, and selection of features
+Existing VEDA instances:
+- Template
+- Earthdata
+- Greenhouse Gas Center Portal
+- earth.gov
+Planned VEDA instances:
+- Air Quality Portal
+
+### 7. There are a wide variety of potential users of the VEDA Dashboard, although each science team usually narrows down on one or two groups for their instance
+<Brian’s graphic of different target users>
+
+### 8. VEDA UI provides a suite of tools, widgets, and pages for science teams to create their own websites, rather than a replicable white-label website/dashboard
```

Comment:
```suggestion
### How can I use VEDA UI to power a web presence?

_VEDA UI provides a suite of tools, widgets, and pages for science teams to create their own websites, rather than a replicable white-label website/dashboard_
```